todo_app/tests_e2e/test_selenium.py

```diff
@@ -12,9 +13,28 @@
 
 @pytest.fixture()
 def driver():
-    with webdriver.Firefox() as driver:
+    #with webdriver.Firefox() as driver:
+    #    yield driver
```

Comment:
Delete commented out code.

Also, having switched from Firefox to Chrome I expect the README to need updating. It should mention that to run the e2e tests you need to have Chrome and chromedriver